apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/spi/oidc/OidcLocalUserServiceImpl.java

```diff
@@ -87,7 +87,7 @@ public void updateUserInfo(UserInfo newUserInfo) {
   }
 
   @Override
-  public List<UserInfo> searchUsers(String keyword, int offset, int limit) {
+  public List<UserInfo> searchUsers(String keyword, int offset, int limit, boolean includeInactiveUsers) {
```

Comment:
The oidc implementation is almost the same as spring security implementation, I believe we could also add the logic to includeInactiveUsers? cc @vdisk-group